hw11/tests/test_hw11_t01.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from hw11.hw11_t01 import SimplifiedEnum
+
+
+def test_colorsenum_attributes():
+    class ColorsEnum(metaclass=SimplifiedEnum):
+        __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+
+    assert ColorsEnum.RED == "RED"
+    assert ColorsEnum.ORANGE != "BLACK"
```

Comment:
Orange is definitely could be red. Why do you test it for blackness?

Reply:
This comment is about the fact that test with != equal does not make sense.  ORANGE is not equal RED, orange is not equal DOOR, it does not equal BLACK as well. The only thing in the universe that does not not-equals to ORANGE is ORANGE. So instead of picking a random thing of the infinite set of possibilities, just check that it is ORANGE.

Some for other attributes. I want you to test BLACK. And BLUE.